src/status_im2/contexts/activity_center/notification/mentions/view.cljs

```diff
@@ -0,0 +1,48 @@
+(ns status-im2.contexts.activity-center.notification.mentions.view
+  (:require [clojure.string :as string]
+            [i18n.i18n :as i18n]
+            [quo.components.animated.pressable :as animation]
```

Comment:
I wonder the same thing @J-Son89. Another approach is to keep track in a single issue, whatever is easier for us to keep up to date.

If we run the following command, we can see all imports to old stuff in the new `src/status_im2` directory. The same kind technique can be used in other directories to quickly pinpoint what's missing.

```bash
rg -e '\[quo\..*' src/status_im2
```

Output:

```
src/status_im2/setup/events.cljs
10:            [quo.theme :as quo.theme]

src/status_im2/subs/chat/chats.cljs
5:            [quo.design-system.colors :as colors]

src/status_im2/contexts/communities/requests/actions/view.cljs
6:            [quo.react-native :as rn]

src/status_im2/contexts/quo_preview/community/community_list_view.cljs
2:  (:require [quo.react-native :as rn]
3:            [quo.previews.preview :as preview]

src/status_im2/contexts/quo_preview/community/data.cljs
3:   [quo.design-system.colors :as quo.colors]

src/status_im2/contexts/quo_preview/community/community_membership_list_view.cljs
2:  (:require [quo.react-native :as rn]
3:            [quo.previews.preview :as preview]

src/status_im2/contexts/quo_preview/community/token_gating.cljs
2:  (:require [quo.previews.preview :as preview]
3:            [quo.react-native :as rn]

src/status_im2/contexts/quo_preview/community/discover_card.cljs
2:  (:require [quo.react-native :as rn]
3:            [quo.previews.preview :as preview]

src/status_im2/contexts/quo_preview/community/community_card_view.cljs
7:            [quo.design-system.colors :as quo.colors]

src/status_im2/contexts/quo_preview/selectors/selectors.cljs
4:            [quo.previews.preview :as preview]
5:            [quo.react-native :as rn]

src/status_im2/contexts/quo_preview/posts_and_attachments/messages_skeleton.cljs
2:  (:require [quo.react-native :as rn]

src/status_im2/contexts/activity_center/view.cljs
3:            [quo.components.safe-area :as safe-area]
4:            [quo.react :as react]

src/status_im2/contexts/activity_center/notification/mentions/view.cljs
4:            [quo.components.animated.pressable :as animation]

src/status_im2/contexts/chat/messages/list/view.cljs
18:            [quo.react-native :as quo.react]))

src/status_im2/contexts/activity_center/notification/contact_request/view.cljs
3:            [quo.components.animated.pressable :as animation]
```

Reply:
sure 1 issue is fine, but let's ensure whoever does it does so with small p.r's doing one task at a time 🙏

Reply:
Absolutely :+1: And maybe it's better to delay these refactorings so we can reformat the codebase first (next Tuesday it seems).

@siddarthkay, if you want, I can create an issue for us to keep track of progress.

Reply:
@ilmotta : yep go ahead

Reply:
I don't think an issue like this exists on github yet, and even if there are duplicates we can deal with that later

Reply:
Awesome then, will create later today.

Reply:
Guys, here's the new issue https://github.com/status-im/status-mobile/issues/14559, of course a WIP given the amount of stuff to do, but it's a good start. @flexsurfer, I believe you have some useful notes about what's left to do. I'd appreciate if you could add anything in you have in mind to the description, otherwise you can also send me some raw notes and I update the issue for us all.